code/game/gamemodes/cult/cult_items.dm

```diff
@@ -100,12 +105,18 @@
 	icon_state = "cult_hoodalt"
 	item_state = "cult_hoodalt"
 
+/obj/item/clothing/head/culthood/alt/ghost
+	flags = NODROP|DROPDEL
+
 /obj/item/clothing/suit/cultrobes/alt
 	name = "cultist robes"
 	desc = "An armored set of robes worn by the followers of Nar-Sie."
 	icon_state = "cultrobesalt"
 	item_state = "cultrobesalt"
 
+/obj/item/clothing/suit/cultrobes/alt/ghost
```

Comment:
should set some cool ```user_vars_to_edit``` on this (eg: colouring the mob slightly red, slightly see through, etc.)